bluesky/run_engine.py

```diff
@@ -508,7 +508,13 @@ def __call__(self, plan, subs=None, *, raise_if_interrupted=False,
         if not self.state.is_idle:
             raise RuntimeError("The RunEngine is in a %s state" % self.state)
 
-        futs = [f for sup in self.suspenders for f in sup.get_futures()]
+        futs = []
+        for sup in self.suspenders:
+            f_lst, msg = sup.get_futures()
+            if f_lst:
+                futs.extend(f_lst)
+                print(msg)
```

Comment:
@klauer @tacaswell Is it intention that this message does not print when the suspender trips mid-scan -- only when it trips at the start? Also, should the message indicate that the plan will still run if/when the suspender becomes un-tripped?